packages/functions/test/oidc/awsCredentialsProvider.test.ts

```diff
@@ -0,0 +1,48 @@
+import { expect, it, vi } from 'vitest';
+import { awsCredentialsProvider } from '../../src/oidc';
+
```

Comment:
<!-- Suggestion comments propose improvements to the code. Mention specifically what should be changed and what the benefit would be.  -->
**suggestion (soft-blocking):** add single root `describe`

Can you wrap all of this in a single, root `describe`?